test/cucumber/Conferencia.feature

```diff
@@ -144,4 +144,64 @@ Feature: conferencia
     And I select the option Serach for Conference at the conference page
     Then a list of all conferences containing that date will be presented in the conference screen
 
-# voces podem criar cenários para ordenar a lista de conferencia, filtrar a lista,  verificar se alguns campos podem ser opcionais, etc.
+  Scenario: Publish a new article  
+    Given I am at the article registration page  
+    When I am filling in the author field
+    And As I type the name, they come up suggestions of names containing the string entered as "And" may appear names like " Anderson " or " Candido "
```

Comment:
Reescrever usando exemplo apenas. Inclusive, as sugestões do sistema vão depender dos autores previamente cadastrados, né? Isso tem que ficar claro. Por exemplo, se existirem os autores "Anderson" e "Candido", quando o usuário digitar "and" ambos os nomes poderão ser sugeridos.